modulemd/v2/modulemd-v2-docs.xml

```diff
@@ -39,6 +39,8 @@
         <xi:include href="xml/modulemd-defaults.xml"/>
         <xi:include href="xml/modulemd-defaults-v1.xml"/>
         <xi:include href="xml/modulemd-dependencies.xml"/>
+        <xi:include href="xml/modulemd-module-stream.xml"/>
```

Comment:
I believe this may be missing the `v1`.

Reply:
Looks like it, yeah. Odd, I could have sworn I checked that...